.github/workflows/ci-ubuntu.yml

```
@@ -12,7 +12,7 @@ on:
 ########################################################################
 
 env:
-  IDRIS2_COMMIT: a28bc655440e963591aaa7cfc47adefc4e785452
+  IDRIS2_COMMIT: 60d597fccddcbcc26aa08efd7e93acc50b81f554
```

I also had to migrate to the new version in my PR: https://github.com/frex-project/idris-frex/pull/12/files#diff-87a770b6884e8117dd199cce57508edb045d00580618a51da4763186cf2df461

I think it's probably better to target a released version rather than a specific
commit as long as we can afford to!

```suggestion
  IDRIS2_COMMIT: tags/v0.4.0
```